flow/project.py

```diff
@@ -4911,17 +4895,11 @@ class MyProject(FlowProject):
                 default=0,
                 help="Increase output verbosity.",
             )
-            _parser.add_argument(
-                "--show-traceback",
```

Comment:
Do we need to continue accepting this option and show a deprecation for one release? (I'm fine with just removing it and noting that in the changelog, but it could break user scripts that call with `--show-traceback`.)

Reply:
Just added a `FutureWarning`.